src/GrasshopperTeklaDrawingLink/Components/Text/Arrow component.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Grasshopper.Kernel;
+
+using GTDrawingLink.Properties;
+using GTDrawingLink.Tools;
+using GTDrawingLink.Types;
+
+using System.Collections.Generic;
+using System.Drawing;
+
+using Tekla.Structures.Drawing;
+
+namespace GTDrawingLink.Components.Text {
+    public class ArrowAttributeComponent : TeklaComponentBase {
```

Comment:
Please fix file name. _Arrow component.cs_ does not meet _ArrowAttributeComponent_ class name.